src/platform/forms/save-in-progress/actions.js

```diff
@@ -399,22 +417,15 @@ export function fetchInProgressForm(
 
 export function removeInProgressForm(formId, migrations, prefillTransformer) {
   return (dispatch, getState) => {
-    const trackingPrefix = getState().form.trackingPrefix;
+    const { trackingPrefix } = getState().form;
 
     // Update UI while we’re waiting for the API
     dispatch(setStartOver());
 
     return removeFormApi(formId)
-      .catch(res => {
```

Comment:
So why is this error handling being removed? Is it being handled elsewhere?

Reply:
It is being handled in thte `save-in-progress/api.js` as the first `.catch`